apps/docs/guides/jssg/quickstart.mdx

```diff
@@ -0,0 +1,142 @@
+---
+title: "Quickstart"
+---
+
+Build a working JavaScript ast-grep (jssg) codemod in minutes.
+
+## What is jssg?
+
+JavaScript ast-grep (jssg) is a codemod framework that lets you write code transformations using JavaScript/TypeScript. Instead of complex AST manipulation, you use patterns to find and replace code structures.
+
+**How it works:**
+- Your transform runs in a **QuickJS** runtime with **AWS LLRT** module infrastructure
```

Comment:
Let's say something like this:

jssg is a secure JavaScript runtime similar to Node.js with almost all of the built-in nodejs modules and globals e.g fs, process, etc

Then in perhaps a footnote-like "learn more" section we can say it's p powered by QuickJs for the JS engine and uses LLRT for node modules compatibility, and oxc for module resolution.

Reply:
We should also say it comes with ast grep as a built in module too.